src/sentry/sentry_metrics/indexer/postgres.py

```diff
@@ -0,0 +1,79 @@
+from collections import defaultdict
+from typing import Dict, List, Optional, Set
+
+from sentry.sentry_metrics.indexer.models import MetricsKeyIndexer
+from sentry.utils.services import Service
+
+
+class Record:
+    organization_id: int
+    string: str
+    id: int
+
+
+class PGStringIndexer(Service):  # type: ignore
+    """
+    Provides integer IDs for metric names, tag keys and tag values
+    and the corresponding reverse lookup.
+    """
+
+    __all__ = ("record", "resolve", "reverse_resolve", "bulk_record")
+
+    def _bulk_record(self, org_id: int, unmapped_strings: Set[str]) -> List[Record]:
+        records = []
+        for string in unmapped_strings:
+            obj = MetricsKeyIndexer.objects.create(organization_id=org_id, string=string)
```

Comment:
Do we have a way to create objects in batch? It should be more efficient for postgres.

Reply:
I wasn't using `bulk_create` before because it doesn't call `save()` (which I was using to set the value) but now that I've gone back to using an auto increment field I think we can use that.